R/env.R

```diff
@@ -54,13 +54,13 @@
 #' env(fn)
 #'
 #'
-#' # new_env() creates by default an environment whose parent is the
```

Comment:
ok. I think this should be handled in another PR with a new interface for base types ctors, and look like this: `env(a = obj1, b = obj2)`. Merging this for now.